core/include/kernel/dt.h

```diff
@@ -267,13 +267,14 @@ struct dt_descriptor *get_external_dt_desc(void);
 /*
  * init_external_dt() - Initialize the external DTB located at given address.
  * @phys_dt:	Physical address where the external DTB located.
+ * @dt_sz:	Maximum size of the external DTB.
  *
  * Initialize the external DTB.
  *
  * 1. Add MMU mapping of the external DTB,
  * 2. Initialize device tree overlay
  */
-void init_external_dt(unsigned long phys_dt);
+void init_external_dt(unsigned long phys_dt, size_t dt_sz);
```

Comment:
Please move the changes to dt.h and dt.c to a separate commit. The call sites need to be updated too, but only with the minimal, that is `CFG_DTB_MAX_SIZE` as the new argument.

Reply:
Done.